web app/src/test/java/de/tinf15b4/kino/data/SearchServiceTest.java

```diff
@@ -0,0 +1,83 @@
+package de.tinf15b4.kino.data;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+import java.util.Locale;
+import java.util.Random;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.github.javafaker.Faker;
+import com.google.common.collect.Lists;
+
+import de.tinf15b4.kino.data.cinemas.Cinema;
+import de.tinf15b4.kino.data.cinemas.CinemaRepository;
+import de.tinf15b4.kino.data.movies.AgeControl;
+import de.tinf15b4.kino.data.movies.Genre;
+import de.tinf15b4.kino.data.movies.Movie;
+import de.tinf15b4.kino.data.movies.MovieRepository;
+import de.tinf15b4.kino.data.search.SearchResult;
+import de.tinf15b4.kino.data.search.SearchService;
+
+@RunWith(SpringRunner.class)
```

Comment:
We need some sort of `@RunWith`, otherwise the `@Mock...` and `@Inject...` annotations are not evaluated and the whole thing crashes with a NullPointerException. It might work in Eclipse, but it will not work on the command line or in Jenkins. Trust me, I just tried it.